android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -68,6 +78,88 @@
   Map<String, Object> constants;
   AsyncTask<Void, Void, Map<String, Object>> futureConstants;
 
+  private boolean isDebug = true;
+
+  private boolean isTelephony = true;
+
+  private boolean isCheckPackage = true;
+
+  private List<String> mListPackageName = new ArrayList<>();
+
+  private static final String IP = "10.0.2.15";
+
+  private static final int MIN_PROPERTIES_THRESHOLD = 0x5;
+
+  private static final String[] DEVICE_IDS = {
+          "000000000000000",
+          "e21833235b6eef10",
+          "012345678912345"
+  };
+
+
+  private static final String[] IMSI_IDS = {
+          "310260000000000"
+  };
+
+  private static final String[] GENY_FILES = {
+          "/dev/socket/genyd",
+          "/dev/socket/baseband_genyd"
+  };
+
+  private static final String[] ANDY_FILES = {
+          "fstab.andy",
+          "ueventd.andy.rc"
+  };
+
+  private static final String[] NOX_FILES = {
+          "fstab.nox",
+          "init.nox.rc",
+          "ueventd.nox.rc"
+  };
+
+  private static final String[] QEMU_DRIVERS = {"goldfish"};
+
+  private static final String[] PIPES = {
+          "/dev/socket/qemud",
+          "/dev/qemu_pipe"
+  };
+
+  private static final String[] PHONE_NUMBERS = {
+          "15555215554", "15555215556", "15555215558", "15555215560", "15555215562", "15555215564",
+          "15555215566", "15555215568", "15555215570", "15555215572", "15555215574", "15555215576",
+          "15555215578", "15555215580", "15555215582", "15555215584"
+  };
+
+  private static final String[] X86_FILES = {
+          "ueventd.android_x86.rc",
+          "x86.prop",
+          "ueventd.ttVM_x86.rc",
+          "init.ttVM_x86.rc",
+          "fstab.ttVM_x86",
+          "fstab.vbox86",
+          "init.vbox86.rc",
+          "ueventd.vbox86.rc"
+  };
+
+  private static final Property[] PROPERTIES = {
```

Comment:
These are called "PROPERTIES", but they are only used for QEmu checking, should be QEMU_PROPERTIES maybe?

Reply:
removed
https://github.com/react-native-community/react-native-device-info/pull/721/commits/da3c60a87e5b883e385f4a5d8d43092f34e78b52